dom.bs

```diff
@@ -1201,6 +1201,9 @@ for discussion).
  <a>relatedTarget</a> against <var>target</var> if <var>event</var>'s <a>relatedTarget</a> is
  non-null, and null otherwise.
 
+ <li><p>If <var>event</var>'s {{Event/isTrusted}} attribute is true and <var>target</var> and
+ <var>relatedTarget</var> are identical, terminate these steps.
```

Comment:
Is that okay to terminate these steps without a return value?
That would be no difference in practice, but I am not sure what is the best.